README.md

```diff
@@ -26,20 +26,20 @@ This project is nascent and still under heavy development, but first steps:
 
 1.  Ensure you have [rustup](https://www.rustup.rs/) installed.
 
-1.  Clone the repository:
+2.  Clone the repository:
 
     ```shell
     git clone https://github.com/tock/libtock-rs
     cd libtock-rs
     ```
 
-1.  Install the dependencies:
+3.  Install the dependencies:
 
     ```shell
     make setup
     ```
 
-1.  Use `make` to compile and run an example app.
+4.  Use `make` to build examples
```

Comment:
I'm not sure I like this renumbering. Using `1.` for all the numbers means that if we add or remove a step in the middle, the rest are renumbered automatically.

Reply:
Oh, I will fix the numbers back to `1.`